<?php

namespace App\Exceptions;

use Exception;
use Illuminate\Http\Response;

class CpfExisteException extends Exception
{
    protected $message = 'CPF informado jÃ¡ existe!';
    protected $code = Response::HTTP_BAD_REQUEST;

    public function render($request)
    {
        return response()->json(['mensagem' => $this->getMessage(), "status" => $this->getCode()], $this->getCode());
    }
}
